tests/emails/offerer_booking_recap_test.py

```diff
@@ -136,7 +141,7 @@ def test_should_not_truncate_price(self, app):
         stock = create_stock_from_offer(event_offer, beginning_datetime=beginning_datetime, price=5.86, available=10)
         booking = create_booking(user=user, stock=stock, venue=venue, token='ABC123')
         stock.bookings = [booking]
-        recipient = ['dev@passculture.app']
+        recipient = ['dev2@example.com']
```

Comment:
```suggestion
        recipient = ['initial_recipient@example.com']
```
je trouve ça un peu confusant ce dev2@example.com en fait c'est le destinataire normal, sauf que là son mail est écrasé par DEV_EMAIL_ADDRESS ?

(si tu fais la modif il faut la faire en dessous aussi)